vars.mk

```diff
@@ -13,11 +13,11 @@
 #   limitations under the License.
 
 # Pinned Versions
-GO_VERSION=1.16.3-alpine
-CLI_VERSION=20.10.2
-ALPINE_VERSION=3.12.2
-GOLANGCI_LINT_VERSION=v1.33.0-alpine
-GOTESTSUM_VERSION=0.6.0
+GO_VERSION=1.18-alpine
+CLI_VERSION=20.10.12
```

Comment:
While we're updating; could you make this 20.10.14 (which is latest)? (same for the Dockerfile)